File: Mathematics/CountDigits.java
import java.util.*;

class CountDigits{

     static int countDigits(int x){

        int digits =0;
        while(x>0){
            x=x/10;
            digits++;
        }
        return digits;
    }

    public static void main(String[] args) {
        int number = 2506;
        System.out.println(countDigits(number));
    }
}

File: Mathematics/CountDigits.cpp
#include<iostream>
using namespace std;

int countDigits(int x){
    int digits=0;
    while(x>0){
        x=x/10;
        digits++;
    }

    return digits;
}

int main(){

    int number = 2504;
    cout<<countDigits(number);

    return 0;
}